### PR TITLE
feat: configure default object/primitive value creation in form editor

### PR DESCRIPTION
**Pull request by erdkse**
This PR adds two new settings for controlling Form Editor behavior when it comes to automatic creation of default values - see linked issue

## Changes

- Adds new settings

## Fixes

-

## How to test it

-

## Screenshots

image

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/redux/services/projectConfig.ts [hunk 1]
```diff
@@ -52,6 +52,8 @@ export const populateProjectConfigToWrite = (state: AppConfig | SerializableObje
     kustomizeCommand: state.projectConfig.settings.kustomizeCommand,
     hideExcludedFilesInFileExplorer: state.projectConfig.settings.hideExcludedFilesInFileExplorer,
     enableHelmWithKustomize: state.projectConfig.settings.enableHelmWithKustomize,
+    createDefaultObjects: state.projectConfig.settings.createDefaultObjects,
+    setDefaultPrmitiveValues: state.projectConfig.settings.setDefaultPrmitiveValues,
```
Comment (olensmar) [inline src/redux/services/projectConfig.ts:56]: typo: setDefaultPrimitiveValues

**Review by olensmar — APPROVED** [review]
LGTM - but I'm biased :D

**Review by devcatalin — APPROVED** [review]
LGTM

**Comment by kingdonb** [comment]
Thank you, I had a quick try with 1.5.0 and tried some of the things that led me to comment on #1222 origiinally - I haven't been in to change settings yet, because I'm happy with the defaults and nothing surprising at all seems to be happening. 👍

So I'd say it landed successfully! TYVM

**Comment by olensmar** [comment]
fantastic - thanks for giving it a try @kingdonb and letting us know.. 

We're planning the next release now and would like to look into supporting Flux more natively - perhaps something to discuss at some point!?

**Pull request merged**